content/blog/hackaton-2022-building-a-pong.md

```diff
@@ -0,0 +1,212 @@
+
+---
+Title: Hackapong fun at the shop!
+Date: 2022-11-04 13::55:00
+Author: Jean-Philippe Lamebrt - alex Hoguin
+Category: WDA
```

Comment:
Should we talk about WDA ?

Reply:
I dont know . I mean ... What are we going to talk about if we are not talking about WDA.